src/pages/preferences/preferencesTabs/cliTab/DownloadCard.tsx

```diff
@@ -139,25 +143,30 @@ const prettyDisplayNameAccordion = {
 };
 
 const filterBinaries = (binary: ClientBinary) =>
-  /darwin_amd64\/|linux_amd64\/|windows_amd64\//.test(binary.url);
+  /darwin_arm64\/|darwin_amd64\/|linux_amd64\/|windows_amd64\//.test(
+    binary.url
+  );
+
+// @ts-expect-error
+const Container = styled(Card)`
+  padding: 24px;
+` as typeof Card;
 
-const Container = styled(SiderCard)`
-  padding-left: 20px;
-  padding-top: 20px;
-  padding-bottom: 20px;
-`;
 const CardGroup = styled.div`
   display: flex;
+  align-items: space-between;
 `;
-const CliDownloadCard = styled(SiderCard)`
+
+// @ts-expect-error
+const CliDownloadCard = styled(Card)`
   display: flex;
   flex-direction: column;
-  width: 180px;
-  padding-top: 20px;
-  padding-bottom: 20px;
-  padding-left: 20px;
-  margin-right: 16px;
-`;
+  justify-content: space-between;
+  padding: 16px;
+  margin-right: 8px;
+  margin-bottom: 16px;
```

Comment:
nit: I'd put this `margin-bottom` styling on the `CardGroup` rather than on each individual card